# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/kasc/openal/alsa_header.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---


Some ALSA include corrections ...

  - Rene Rebe <rene@exactcode.de>

--- openal-2003-05-09/linux/src/arch/alsa/alsa.c.orig	2002-09-20 19:47:31.000000000 +0200
+++ openal-2003-05-09/linux/src/arch/alsa/alsa.c	2004-05-11 08:50:34.670020144 +0200
@@ -21,7 +21,9 @@
 #include "arch/interface/interface_sound.h"
 #include "arch/alsa/alsa.h"
 
+#include <time.h>
 #include <alsa/asoundlib.h>
+#include <alsa/pcm.h>
 
 #include "al_config.h"
 #include "al_debug.h"
